CMakeLists.txt

```diff
@@ -40,3 +40,6 @@ endif()
 
 # STMEMSC - Hardware Abstraction Layer for ST sensor
 add_subdirectory_ifdef(CONFIG_HAS_STMEMSC sensor/stmemsc)
+
+# Add BT firmware images upgrade support
+add_subdirectory_ifdef(CONFIG_BT ble_firmware)
```

Comment:
We need a more specific option. BT could be enable on a nordic chip which uses this repo for sensors

Reply:
Yes, sure. I also now recalled that I have to separate the definitions in CMakeLists.txt for the images of different boards